Dispatchfile

```diff
@@ -0,0 +1,72 @@
+#!starlark
+# vi:syntax=python
+#
+
+git = "src-git"
+
+def secretVar(name, key):
+    return k8s.corev1.EnvVarSource(secretKeyRef=k8s.corev1.SecretKeySelector(localObjectReference=k8s.corev1.LocalObjectReference(name=name), key=key))
+
+gitResource(git,
+    url="$(context.git.url)",
+    revision="$(context.git.commit)")
+
+dindTask("publish", inputs=[git], steps=[
+  v1.Container(
+    name = "publish-charts",
+    image = "mesosphere/dispatch-dind:mesosphere-charts",
```

Comment:
Where does this image come from? Would you mind adding a link and explanation here so that we know where to look? I personally would prefer to have the Dockerfile and build process for it in this repo so everything is self-contained.

Reply:
@makkes good catch, I actually forgot to remove that line!